Digizuite.Test/IntegrationTests/UploadServiceTest.cs

```diff
@@ -49,17 +49,18 @@ public async Task CanUploadFile()
         public async Task CanReplace()
         {
             var service = ServiceProvider.GetRequiredService<IUploadService>();
+            var assetId = 257;
 
             var file = new FileInfo(TestFileName);
 #pragma warning disable CA2000
             await using var stream = new FileStream(file.FullName, FileMode.Open, FileAccess.Read);
 #pragma warning restore CA2000
             var listener = new SimpleUploadProgressListener();
-            var uploadResponse = await service.Replace(stream, "replace-from-unit-test.png", "unittest", 257,
+            var uploadResponse = await service.Replace(stream, "replace-from-unit-test.png", "unittest", assetId,
                 KeepMetadata.Keep, Overwrite.AddHistoryEntry, listener).ConfigureAwait(false);
 
             Assert.That(uploadResponse.ItemId, Is.EqualTo(listener.FinishedItemId));
-            Assert.That(uploadResponse.AssetId, Is.EqualTo(56));
+            Assert.That(uploadResponse.AssetId, Is.EqualTo(assetId));
```

Comment:
These asset id's are intentionally different. Yes, I know that is slightly confusing.

Reply:
Test is failing because of this, is it smth with test or sevice?

Reply:
I believe it's because of the service itself having a bad day. The unit test dam environment is quite unstable :/

Reply:
Yeah, so this was apparently an error from a merge. What you actually did here was correct.